### ISSUE TITLE
Cubical dd-dd-term should default to True

### ISSUE THREAD

**Issue by bennahugo** [SpheMakh/Stimela#355]
@SpheMakh Currently the dd labeled gains are not marked as direction dependent. This probably explains why the calibration doesn't improve:
```
  Name         dd-dd-term
  Description  Determines whether this term is direction dependent. --model-ddes must
  Type         bool
  Default      None
```

**Comment by bennahugo** [comment]
@KshitijT take note

**Comment by o-smirnov** [comment]
And make sure you have ``--model-ddes auto``. That is the default and shouldn't need changing.

**Comment by SpheMakh** [comment]
Fixed in a4e4089

**Comment by SpheMakh** [comment]
@o-smirnov done in 40df370
